DEVELOPING_OPAL/validate/src/test/java/org/opalj/fpcf/properties/linear_constant_propagation/lcp/ConstantValue.java

```diff
@@ -18,9 +18,9 @@
 @Retention(RetentionPolicy.CLASS)
 public @interface ConstantValue {
     /**
-     * The name of the variable
+     * The PC of this variable in the TAC
```

Comment:
PCs are not a property of the TAC, but of the bytecode. Do not confuse them with TAC indices.